internal/backend/runtime/omni/controllers/omni/machine_set_status.go

```diff
@@ -195,6 +196,12 @@ func (handler *machineSetStatusHandler) reconcileRunning(ctx context.Context, r
 
 	requeue, err := handler.reconcileMachines(ctx, r, logger, rc)
 	if err != nil {
+		// qtransform controller ignores PhaseConflict errors
+		// mask the error and make the controller fail explicitly instead
+		if state.IsPhaseConflictError(err) || state.IsConflictError(err) {
+			return fmt.Errorf("operations failed %s", err.Error())
```

Comment:
intended, or?
```suggestion
			return fmt.Errorf("operations failed: %w", err)
```

Reply:
nvm, didn't pay a attention to the comment above.

Reply:
intended, see the comment above

Reply:
I think we kept ignoring `PhaseConflictError` e.g. when you try update something while it is being torn down, that was I guess for primary output only, but since that time we got extra outputs as well

so idk how we can best fix that on the COSI side - enhance the error to track  which resource it was?

(I guess here it's an extra output)

Reply:
Yeah, adding the resource ref to the error can work I guess.

Reply:
https://github.com/cosi-project/runtime/pull/492